src/VirtoCommerce.Platform.Web/Security/ApplicationBuilderExtensions.cs

```diff
@@ -38,6 +43,27 @@ public static IApplicationBuilder UseSecurityHandlers(this IApplicationBuilder a
             return appBuilder;
         }
 
+        /// <summary>
+        /// Schedule a periodic job for prune expired/invalid authorization tokens
+        /// </summary>
+        /// <param name="appBuilder"></param>
+        /// <returns></returns>
+        public static IApplicationBuilder UsePruneExpiredTokensJob(this IApplicationBuilder appBuilder)
+        {            
+            var recurringJobManager = appBuilder.ApplicationServices.GetService<IRecurringJobManager>();
+            var settingsManager = appBuilder.ApplicationServices.GetService<ISettingsManager>();
+
+            RecurringJobExtensions.WatchJobSetting(recurringJobManager,
```

Comment:
please change to recurringJobManager.WatchJobSetting(settingsManager, ...

Reply:
Agree!